testcontainers/arangodb.py

```diff
@@ -0,0 +1,64 @@
+"""
+ArangoDB container support.
+"""
+from os import environ
+from testcontainers.core.generic import DbContainer
+from testcontainers.core.waiting_utils import wait_for_logs
+
+
+class ArangoDbContainer(DbContainer):
+    """
+    ArangoDB container.
+
+    Example
+    -------
+    The example will spin up a ArangoDB container.
+    You may use the :code:`get_connection_url()` method which returns a arangoclient-compatible url
+    in format :code:`scheme://host:port`. As of now, only a single host supported (over HTTP).
+    ::
+
+        with ArangoContainer("arangodb:3.9.1") as arango:
+            client = ArangoClient(hosts=arango.get_connection_url())
+
+            # Connect
+            sys_db = arango_client.db(username='root', password='')
+
+            # Create a new database named "test".
+            sys_db.create_database("test")
+    """
+    def __init__(self, image="arangodb:latest", **kwargs):
+        super().__init__(image=image)
+        self.port_to_expose = 8529
+        self.with_exposed_ports(self.port_to_expose)
+
+        self.arango_host = kwargs.get('ARANGO_HOST', 'localhost')
+
+        # https://www.arangodb.com/docs/stable/deployment-single-instance-manual-start.html
+        self.arango_no_auth = kwargs.get(
```

Comment:
Shall we also move these to proper keyword arguments? That should make it a bit easier for code completion, for example. Something like this?

```python
# arango_no_auth=None in the __init__ signature.
self.arango_no_auth = arango_no_auth or os.environ.get("ARANGO_NO_AUTH")
```

Reply:
Makes sense. 👍 We should probably update that in mysql at some point.